Unit_Tests/test_CTP_generator.py

```diff
@@ -1,41 +1,58 @@
 import jax
 import jax.numpy as jnp
 import sys
-sys.path.append('..')
+
+sys.path.append("..")
```

Comment:
Ideally you can avoid doing this by making the project into a package. Might require a small install script (but you should add one anyway to handle the `requirements.txt`)